chapters/vulkan_profiles.adoc

```diff
@@ -0,0 +1,521 @@
+
+== What are Vulkan Profiles?
+
+A Vulkan Profile is a well-defined set of capabilities, (features, extensions, formats, queue families, etc.) that can be used as a baseline for application development. Profiles are defined in JSON format and can be used to:
+
+* Simplify development by providing a known set of capabilities
+* Ensure compatibility across a range of devices
+* Reduce the need for complex capability checking and fallback code
+* Provide a clear target for both developers and hardware vendors
+* Automatically enable required features and extensions without manual configuration
+
+The Khronos Group maintains a set of official profiles, and vendors can also
+define their own profiles for their specific hardware. Additionally, Vulkan
+application developers can create profiles to specify engine code paths, like
+ Unreal Engine does. Or libraries such as Zink can specify their Vulkan requirements for OpenGL to run, enabling hardware vendors to prioritize their driver development to support specific code paths on engines or libraries.
+
+=== History and Purpose
+
+Vulkan Profiles were introduced to address the challenge of fragmentation in the Vulkan ecosystem. With many different devices supporting various combinations of features and extensions, developers faced difficulties in:
+
+* Determining which features to target for maximum device compatibility
+* Managing fallback paths for devices with limited capabilities
+* Communicating requirements clearly to hardware vendors
+
+Profiles solve these problems by providing a convention to define and reference sets of Vulkan capabilities. They serve as a contract between application developers and hardware vendors, ensuring that devices claiming support for a profile will provide all the capabilities defined in that profile.
+
+=== Relationship with Vulkan Versions
+
+Vulkan Profiles are complementary to Vulkan versions. While Vulkan versions (1.0, 1.1, 1.2, etc.) define the core API functionality, profiles define specific subsets of features, extensions, and limits that may span across different Vulkan versions.
+
+For example:
+
+* The Android Baseline 2022 profile requires Vulkan 1.1 core functionality
+* The LunarG Desktop Baseline 2022 profile for desktop platforms requires Vulkan 1.1 with specific extensions
+* Future profiles may target newer Vulkan versions as they become more widely supported
+
+This approach allows developers to target specific capability sets without being tied to a particular Vulkan version, providing more flexibility in application development.
+
+== Official Profiles
+
+The Khronos Group maintains several profiles, which are defined in the link:https://github.com/KhronosGroup/Vulkan-Profiles/tree/main/profiles[Vulkan-Profiles repository]. The only official profiles are the Khronos Roadmap profiles and the Android Baseline profile. The others are maintained by Google and LunarG:
+
+* *LunarG Desktop Baseline 2022* - A baseline profile for desktop platforms
```

Comment:
Again "LunarG Desktop Baseline 2022" is not an "official" profile, at least it has nothing to do with Khronos.

Then, it's not clear what you meant with "Official"